incubator/nginx-ingress/values.yaml

```diff
@@ -28,6 +28,8 @@ resources:
     cpu: 100m
     memory: 128Mi
 
+proxy: true
```

Comment:
nit: I think `proxy` is overloaded here, `useProxyProtocol` would be more accurate.